pages/instructorAssessmentAccess/instructorAssessmentAccess.ejs

```diff
@@ -6,6 +6,7 @@
   <body>
     <%- include('../partials/navbar'); %>
     <div id="content" class="container-fluid">
+      <%- include('../partials/assessmentSyncErrorsAndWarnings'); %>
```

Comment:
I wonder whether it would be cleaner/easier to put this code in the navbar? That is, I think we want to display the "this assessment is not synced" message on all pages for the assessment? The navbar code is the place where we know which list of pages this is. Otherwise I think we need to add an `include()` like this to essentially all of the instructor pages?

Reply:
Good call, that makes sense to me. I think this code predates some of the navbar refactoring that was done "recently", so that probably didn't occur to me the first time around.